test.sh

```diff
@@ -56,4 +56,4 @@ cd parity-clib-examples/cpp && \
 # Running tests
 echo "________Running Parity Full Test Suite________"
 git submodule update --init --recursive
-time cargo test  $OPTIONS --features "$FEATURES" --all $1 -- --test-threads 8
+time cargo test  $OPTIONS --features "$FEATURES" --all $1 -- --test-threads 4
```

Comment:
why's that?

Reply:
I want to time this to check if the builds went faster. Right now the load average is about 16, which is double the amount of CPU's available. Was hoping reducing the amount of threads would cause less resource contention. But the times look about the same :/